compose-files/docker-compose.yml

```diff
@@ -84,6 +84,13 @@ services:
       FLASK_ENV: development
       FLASK_APP: iib/web/wsgi.py
       IIB_DEV: 'true'
+      OTEL_EXPORTER_OTLP_PROTOCOL: "http/protobuf"
+      OTEL_SERVICE_NAME: "iib-api"
+      OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_REQUEST: "content-type,traceparent,tracestate,custom_response_header"
+      OTEL_INSTRUMENTATION_HTTP_CAPTURE_HEADERS_SERVER_RESPONSE: "content-type,traceparent,tracestate,custom_response_header"
+      OTEL_PROPAGATORS: "tracecontext,baggage"
+      OTEL_EXPORTER_OTLP_TRACES_PROTOCOL: "http/protobuf"
+      OTEL_RESOURCE_ATTRIBUTES: "deployment.environment=tracing-test, service.name=iib"
```

Comment:
For development env, maybe we can look at adding otel collector container?

Reply:
I am not sure I understood how to achieve that. But we will have to inject all the attributes via configmap.

Reply:
I think we can resolve this. @yashvardhannanavati  MR helped answer this.

Reply:
Yash's MRs help for other env (qe, stage, prod) and not local dev containerized env.

Reply:
@zanssa Yes, we should add otel collector container for the dev env. Let's tackle it in a separate issue? I'll file one